Turkey/TestParser.cs

```diff
@@ -32,12 +35,12 @@ public class TestParser
             {
                 case "xunit":
                     test = new XUnitTest(directory, system, nuGetConfig, descriptor, enabled);
-                    return (true, test);
+                    return Task.FromResult((true, test));
                 case "bash":
                     test = new BashTest(directory, system, nuGetConfig, descriptor, enabled);
-                    return (true, test);
+                    return Task.FromResult((true, test));
                 default:
-                    return (false, null);
+                    return Task.FromResult((false, test));
```

Comment:
```suggestion
                    return Task.FromResult((false, null));
```

Reply:
Setting it this way causes an error when compiling. Task.FromResult can't use null as an argument. test is set to null by default so it's the same as if null was used and Task.FromResult accepts test as an argument.

Reply:
This method can probably be made non-async.

> Task.FromResult can't use null as an argument

You mean: it can't figure out what type the `null` should be. You can make that explicit by using `(Test)null` (or `(Test?)null` in nullable enabled code).

Reply:
Ah OK, my mistake.